libs/ballot-interpreter-vx/src/utils/qrcode/with_cropping.ts

```diff
@@ -1,56 +0,0 @@
-import { Rect } from '@votingworks/types';
-import { DetectQrCode, DetectQrCodeResult } from '../../types';
-import { crop } from '../crop';
-import { flipRectVerticalAndHorizontal } from '../geometry';
-
-/**
- * Makes a new QR code detector from an existing one that works by cropping the
- * bottom-right and top-left corners of the image before passing it on.
- */
-export function withCropping(
```

Comment:
This functionality was all duplicated in `services/scan`, but in a totally different way. Fun!